apps/src/codebridge/PopUpButton/PopUpButton.tsx

```diff
@@ -56,9 +57,46 @@ export const PopUpButton = ({
         return newIsOpen;
       });
     },
-    [setIsOpen, setIsOpenFalse]
+    [setIsOpenFalse]
   );
 
+  // Effect to update dropdown position when it is shown.
+  useEffect(() => {
+    const updateDropdownPositionIfShown = () => {
+      if (isOpen) {
+        if (buttonRef && dropdownRef.current) {
+          const dropdownRect = dropdownRef.current.getBoundingClientRect();
+          const buttonRect = buttonRef.getBoundingClientRect();
+          const top = buttonRect.top + buttonRect.height + 5 + window.scrollY;
+          const left =
+            alignment === 'right'
+              ? buttonRect.right - dropdownRect.width + window.scrollX
```

Comment:
This is matching the position in RTL currently, but wondering if we'd like to move the popup options so that they don't cover the files/folders below. Or we could do later!

![Screenshot 2024-11-19 at 12 53 49 PM](https://github.com/user-attachments/assets/b48add1d-59cc-469b-8c83-a3230ed42ad9)

![Screenshot 2024-11-19 at 12 53 39 PM](https://github.com/user-attachments/assets/a9833f2f-cee3-43da-8b17-2494662a88af)

Reply:
I think we will handle that separately, we haven't done rtl work on codebridge yet